rocks-db/src/asset_client.rs

```diff
@@ -113,6 +113,45 @@ impl Storage {
         );
         Ok(())
     }
+
+    pub fn clean_syncronized_idxs_with_batch(
+        &self,
+        asset_type: AssetType,
+        last_synced_key: Vec<u8>,
+    ) -> Result<()> {
+        let (from, cf) = match asset_type {
+            AssetType::Fungible => {
+                let cf = self.fungible_assets_update_idx.handle();
+                let key_type_pairs = self.fungible_assets_update_idx.get_from_start(1);
+
+                if key_type_pairs.is_empty() {
+                    return Ok(());
+                }
+                let from = key_type_pairs[0].0.clone();
+                (from, cf)
+            }
+            AssetType::NonFungible => {
+                let cf = self.assets_update_idx.handle();
+                let key_type_pairs = self.assets_update_idx.get_from_start(1);
+
+                if key_type_pairs.is_empty() {
+                    return Ok(());
+                }
+                let from = key_type_pairs[0].0.clone();
+                (from, cf)
+            }
+        };
+
+        if from == last_synced_key {
+            return Ok(());
+        }
+
+        let mut batch = rocksdb::WriteBatchWithTransaction::<false>::default();
```

Comment:
Can you please explain the reason for using batch here? It seems like you perform a single operation on this batch, so does it make sense to use it at all?

Reply:
Not needed anymore, indeed. Previously it was multiple deletions via iterator